base/sysinfo.jl

```diff
@@ -200,6 +216,11 @@ function cpu_summary(io::IO=stdout, cpu::AbstractVector{CPUinfo} = cpu_info())
     _cpu_summary(io, cpu, first, length(cpu))
 end
 
+"""
+    Sys.cpu_info()
+
+Gather information about all CPUs in the system through a syscall using LibUV.
```

Comment:
libuv is usually in lowercase but I am not sure it is needed in the docstring at all.

We dont have any documentation about the type of struct getting returned `CPUinfo`, so there isn't really any useful thing you can do with the return value here. Arguably this should just not be exported (which is breaking).

Reply:
Should we doc it for now then remove the export in the future?